projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -0,0 +1,170 @@
+
+A few bullet points about my relevant background:
+
+* Author of 16 crypto white papers, the kind that not only bring together focused teams, but also keep teams from overpromising and from running afoul of government regulators. 
+
+* Keynote speaker at over 40 crypto events around the world, including with Vitalik Buterin at Blockchain Futurist in Toronto, a speech that received global television coverage for five minutes on CNBC.
+
+* Organized and chaired over a dozen IPv6 events that brought together representatives of over 40 national governments, over 40 US federal agencies, and hundreds of corporations, universities, standards bodies and nonprofits.
+
+* In order to get IPv6 mandated by the US federal government, I got the chairman of the Government and Reform Committee to hold hearings on Internet Leadership. I chose the industry panel of expert witnesses and the government panel, testified, and got acceptance by the CTO of the federal government of my package of policy proposals submitted along with my testimony. As far as I know, this is the only time someone has been able to get a technology or telecommunications protocol mandated by the US federal government. 
+
+* I’ve won four global awards for leadership and community building of new technology innovations and telecommunications protocols, including the first Economist magazine Reader’s Award on behalf of 4G wireless broadband, after a five month vote open to the entire world, vs. Elon Musk and five other innovators.
+
+* Almost 1,000 appearances on television, including 150 times on Science Channel programs.
+
+I will be working with my colleagues Josh Berger and Jesse Berger. Josh has 10+ years in marketing and web development. He has an international team of developers willing to work for PKT Cash to create a dynamic, designed, and in-depth website to host the content and white paper. 
+
+Jesse has 15+ years experience in brand creation and development as well as copywriting. The brands he has created are Radical Studio, including its publishing division and IP comic book studio with over 40 titles with 23 published works, including Hercules, and the feature film Oblivion. Jesse also created Pet Hemp Company and Mantle Networks, to name a few. Jesse has a deep knowledge of the PKT project which will be helpful in crafting communications that will be taken seriously in both crypto and traditional industries.
+## Project deliverables
+The key deliverable will be the white paper, copywriting, digital assets, branding, and other web deliverables and materials needed to explain the project to average consumers and potential community members. Additionally, the deliverables include a dynamic website to host the content.
+
+### White Paper
```

Comment:
1. What is the scope of the white paper?
2. Will it include technical details of both cjdns and the PKT blockchain?

Reply:
1. The scope of the white paper is to further detail the PKT blockchain. The current information on PacketCrypt on Github is dense and geared to a technologist demographic. We want to make the project more approachable to average community members and inspire user adoption. We want to help make this project accessible to all.

2. This project will focus solely on PKT, PacketCrypt, and PKT Cash.

Reply:
I suggest we include a 3-revision clause in the agreement.

Since the white-paper will describe PKT and PKT is managed by the PKT Network Stewards, effectively Alex is suggesting to act like a contractor on behalf of the Network Steward.

If for some reason he doesn't adequately describe all or some of PKT to the Stewart's satisfaction, the Stewart should be able to ask for a limited number of revisions to narrow the gap. The number of revisions should be limited so that the Stewart doesn't waste Alex Lightman's time also on knit-picking.

Reply:
This was in the proposal to have 1-3 revisions, but I further clarified for you.